app/src/main/java/org/apache/fineract/ui/online/DashboardActivity.java

```diff
@@ -166,6 +166,21 @@ public void onBackPressed() {
             new Handler().postDelayed(
                     () -> isBackPressedOnce = false
                     , 2000);
+        } else if (fragment instanceof CustomersFragment) {
+            if (((CustomersFragment) fragment).onBackPressed())
+                super.onBackPressed();
+        } else if (fragment instanceof LedgerFragment) {
+            if (((LedgerFragment) fragment).onBackPressed())
+                super.onBackPressed();
+        } else if (fragment instanceof AccountsFragment) {
+            if (((AccountsFragment) fragment).onBackPressed())
+                super.onBackPressed();
+        } else if (fragment instanceof TellerFragment) {
+            if (((TellerFragment) fragment).onBackPressed())
+                super.onBackPressed();
+        } else if (fragment instanceof ProductFragment) {
+            if (((ProductFragment) fragment).onBackPressed())
+                super.onBackPressed();
```

Comment:
Is it really necessary to add too many if-else conditions here? Can't only one `else block` do the work, as I see whatever fragment it is we are calling `super.onBackPressed()`.

Reply:
I don't know what I had in my mind when I did this. Anyway, will make the changes tomorrow.

Reply:
Hey, Just checked, since I need to cast the fragment to the designated fragment, this is necessary. Most I can do is a `switch` implementation. 
Please suggest if you have any other implementation in mind.

Reply:
Oh Ookay, so then please try `switch` if you can make it simpler with lesser code, otherwise, it looks good to me. :)